test/precompile.jl

```diff
@@ -645,17 +661,17 @@ precompile_test_harness("code caching") do dir
     end
     @test hasspec
     # Test that compilation adds to method roots with appropriate provenance
-    m = which(setindex!, (Dict{M.X,Any}, Any, M.X))
+    m = which(M.store, (M.OldDict{M.X,Any}, Any, M.X))
     @test M.X ∈ m.roots
     # Check that roots added outside of incremental builds get attributed to a moduleid of 0
     Base.invokelatest() do
-        Dict{M.X2,Any}()[M.X2()] = nothing
+        M.store(M.OldDict{M.X2,Any}(), nothing, M.X2())
     end
     @test M.X2 ∈ m.roots
     groups = group_roots(m)
     @test M.X ∈ groups[Mid]           # attributed to M
     @test M.X2 ∈ groups[0]            # activate module is not known
-    @test !isempty(groups[Bid])
+    #@test !isempty(groups[Bid])
```

Comment:
@timholy I've tried to add simplified `OldDict` directly to the test module. Than the code is not in the `Base` any more. Can I remove this test?

Reply:
Maybe replace it by the new module?

Reply:
Thank you, it works.